src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
Use semantic tags where possible. In this case, consider wrapping the `form` elements in a `section` or `main` tag to provide a more semantic structure.